--- a/triton/runtime/_allocation.py
+++ b/triton/runtime/_allocation.py
@@ -1,4 +1,5 @@
 from typing import Optional, Protocol
+import torch
 from contextvars import ContextVar


@@ -17,8 +18,7 @@
 class NullAllocator:

     def __call__(self, size: int, alignment: int, stream: Optional[int]) -> Buffer:
-        raise RuntimeError("Kernel requires a runtime memory allocation, but no allocator was set. " +
-                           "Use triton.set_allocator to specify an allocator.")
+        return torch.cuda.caching_allocator_alloc(size, stream=stream)


 _NULL_ALLOCATOR = NullAllocator()
